### PR TITLE
8100 update seeds

### PR DESCRIPTION
**Pull request by arisah-ijeoma**
this was branched off #488 so it needs to be merged into that branch first before #488 is merged into develop

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -13,11 +13,17 @@
   short_symbol: '£', country_code: 'GB')
 Currency.find_or_create_by(id: 4,name: 'Nicaraguan Cordoba', code: 'NIO', symbol: 'NIC$',
   short_symbol: 'C$', country_code: 'NI')
+Currency.find_or_create_by(id: 5,name: 'Mexican Peso', code: 'MXN', symbol: 'MX$',
```
Comment (emitche) [inline db/seeds.rb]: For all of these currencies, could you add a space between `id: #,` and `name: 'Currency'`?

Comment (arisah-ijeoma) [reply db/seeds.rb]: oops
thanks

**Comment by emitche** [comment]
@arisah-ijeoma Go ahead and integrate this branch into 8116. Thank you!

**Comment by arisah-ijeoma** [comment]
gotcha @emitche

**Pull request merged**